docker-compose.yml

```diff
@@ -18,7 +17,7 @@ services:
       - mysql
     restart: unless-stopped
   mysql:
-    image: mysql-local:1.0.0
+    image: mysql-local:${MYSQL_VERSION:-1.0.0}
```

Comment:
The image name `mysql-local:${MYSQL_VERSION:-1.0.0}` assumes that the `MYSQL_VERSION` environment variable is set. If not, it defaults to `1.0.0`. Ensure that this image tag exists in your Docker registry.